ssr/src/page/notifs/mod.rs

```diff
@@ -0,0 +1,39 @@
+use leptos::*;
+
+use crate::{
+    component::canisters_prov::AuthCansProvider,
+    state::auth::account_connected_reader,
+    utils::{notifications::get_token_for_principal, profile::ProfileDetails},
+};
+
+#[component]
+fn NotifInnerComponent(details: ProfileDetails) -> impl IntoView {
+    let (_, _) = account_connected_reader();
+
+    let on_token_click = move || {
+        let _ = create_local_resource(
```

Comment:
apologies, didn't notice it was a button click

An action is more idiomatic for this
You can create an action, and then dispatch it in the on:click listener

Reply:
created an on_click handler. btw in the real thing we would trigger this through some other action (for eg after video uploaded etc)